resolver-dns-native-windows/src/main/native-package/vs2010.custom.props

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0">
+  <ItemDefinitionGroup>
+      <ClCompile>
+          <TreatWarningAsError>true</TreatWarningAsError>
+      </ClCompile>
+  </ItemDefinitionGroup>
+</Project>
```

Comment:
Do we need this file? Also, I'm apprehensive about treating warnings as errors, since it can break the build down the line as newer compilers add more warnings.

Reply:
No, this is not required. I don't typically write C, so I wanted to be sure that I don't introduce new issues. Thus I decided to treat warnings as errors, because otherwise hawtjni tends to hide them.